libdnf-cli/output/package_list_sections.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "libdnf-cli/output/package_list_sections.hpp"
+
+#include "libdnf-cli/tty.hpp"
+
+#include "libdnf/rpm/nevra.hpp"
+#include "libdnf/rpm/package_set.hpp"
+
+#include <libsmartcols/libsmartcols.h>
+
+#include <algorithm>
+#include <iostream>
+#include <string>
+
+namespace libdnf::cli::output {
+
+PackageListSections::PackageListSections() {
+    table = scols_new_table();
+    scols_table_enable_noheadings(table, 1);
+    if (libdnf::cli::tty::is_interactive()) {
+        scols_table_enable_colors(table, 1);
+    }
+}
+
+
+PackageListSections::~PackageListSections() {
+    scols_unref_table(table);
+}
+
+
+void PackageListSections::print() {
```

Comment:
There is a missing newline at the end of the output for both `dnf5 list` and `dnf5 info`.

Reply:
Yeah, thanks! My shell hides this from me by automatically adding a newline. I'll add a test (as also @kontura suggests).

Reply:
Ah yep didn't see his comment :)